### DISCUSSION TITLE
why does userbot route still exist

### DISCUSSION THREAD

**Discussion by pp0236** [discord-jda/JDA#2037] [Questions and Help]
userbot has removed since 4.2.0 but why it still have client route?

![image](https://user-images.githubusercontent.com/54238880/154903498-4f8749e8-d8d4-4e43-b6eb-fba23232b26a.png)
![image](https://user-images.githubusercontent.com/54238880/154903503-2805e72b-b44c-4ec0-ac0e-a0a5be0fbe99.png)

already removed
![GE9hBiHmgj](https://user-images.githubusercontent.com/54238880/154904208-d1d553a7-45ec-4627-83c8-358541a4bdd1.png)

**Comment by RealYusufIsmail** [comment]
It might be removed in a future pr or might be have been missed. If you want create a pr removing it.
